### ISSUE TITLE
Tag SiteNow sites in Siteimprove

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9410]
# Motivation

It is difficult and time consuming to try and identify compliance issues that need developer investment to resolve for sites that we manage.

Creating a SiteNow tag and tagging all SiteNow sites with this tag would allow us to avoid sorting through any non SiteNow sites and save us time.

# Additional Information

Some comments from our previous discussion:
- Use manifest list to compare and tag sites in Siteimprove
- New SiteNow sites added to Siteimprove should have this tag applied when added
- Devs can add existing tags but not create new

There appears to be existing tags for SNv1, SNv2, SNv3 but no sites using these tags.
- SNv1, SNv2, SNv3 tags have been removed since weren't in use.
- SiteNow tag has been created
- We can discuss if there is a need to recreate and tag SiteNow v2 sites with an additional tag for SiteNow v2.

**Comment by cory-skeers** [comment]
> Look at API to understand if we can rule out:
> * Can we place tags on sites?
> * How quickly could we rule out use of API to pull reporting on WCAG 2.1 A and AA on our sites?

I don't think we can place tags on sites using this, unfortunately. The API is mostly focused on getting info, with a few options to toggle things, request on-demand scans, and creating sites (this could be interesting--maybe an automated or semi-automated process for adding newly created sitenow sites into siteimprove?)

For reporting purposes, the API can be used to pull info on sites including their DCI scores, and narrowed down to a11y by level (A, AA, SI best practice, etc). There are plenty of other options to narrow down even further, like pulling specific issues and/or pulling lists of content which are reporting certain issues.

At least from what I'm seeing, though, the tags aren't available in the API, so any filtering to _our_ sites would have to be done separately (get the manifest, pull the list of siteimprove sites from the API, filter out from the manifest, then pull each site's scores).

I've hopped onto the bandwagon for an API feature request https://portal.feedback.us.pendo.io/app/#/case/472271?cpid=ea78226d-3eb1-42be-bdd2-59510c11e3a9, and am subscribed to it if it gets traction, as inclusion of tags in the API would be a nice help for any reporting.

**Comment by cory-skeers** [comment]
Took an export of sites in siteimprove and a list of domains that have been added in acquia, and did a set comparison to find 87 instances of tagged sites in siteimprove that have _not_ been added to acquia yet, and removed tags from those.

There are another 52 (not .prod) site domains that are in acquia but not siteimprove that I'll check to see if they are in the si-to-add list already, or need to be added yet.

**Comment by cory-skeers** [comment]
Need a bit of work to go through the list of 52 domains. Some may be old inactive sites from prior to our using the internal prod domain that can be removed, some are domains that are in acquia for redirects.

**Comment by cory-skeers** [comment]
24 more sites were added that are live as SiteNow sites but had not been added to Siteimprove. Currently waiting for them to be scanned and become available in siteimprove in order to add users and tagging.
